docs/docs/setup/env-vars.md

```diff
@@ -250,3 +250,29 @@ This can be an absolute path, or relative to main HTML file.
 
 By default the client build will be done to be served with ToolJet server.
 If you intend to use client separately then can set `SERVE_CLIENT` to `false`.
+
+
+#### Tooljet Database (optional) 
+
+PostgREST is a standalone web server that turns your PostgreSQL database directly into a RESTful APIs which is utilized for querying Tooljet Database.
+
+This feature is only enabled if `ENABLE_TOOLJET_DB=` is set to `true`. 
+
+:::tip If you have openssl installed, you can run the following command `openssl rand -hex 32` to generate the value for `PGRST_JWT_SECRET`. 
+
+If this parameter is not specified then PostgREST refuses authentication requests. 
+:::
+
+
+:::info Please make sure that DB_URI is given in the format `postgrest://[PG_USER]:[PG_PASS]@[PG_HOST]/[TOOLJET_DB]` :::
+
+PostgRest database configuration (required)
```

Comment:
PostgREST web server configuration*

Reply:
changed the heading to PostgRest database configuration to PostgREST web server configuration.